Assets/Scripts/PlayerControls.cs

```diff
@@ -148,11 +178,55 @@ void GroundMovement()
 		}
 	}
 
-	Vector3 GetInput()
+	/**
+	 * Raycast forward from the camera's position to detect
+	 * items to interact with.
+	 * 
+	 * This performs RaycastAll, and returns the closest collision.
+	 */
+	bool RaycastForward( out RaycastHit closestHit )
+	{
+		Vector3 camPos = Camera.main.transform.position;
+		Vector3 camForward = Camera.main.transform.forward;
+
+		Debug.DrawRay( camPos, camForward * _interactCheckDistance, Color.yellow, 1.0f, false );
```

Comment:
Yeah, I think it's probably useful. If it get's too annoying we can take it out.